runtime/roots_nat.c

```diff
@@ -218,13 +218,19 @@ void caml_unregister_frametable(intnat *table) {
     d = next_frame_descr(d);
   }
 
-  iter_list(frametables,lnk) {
-    if(lnk->data == table) {
-      previous->next = lnk->next;
-      caml_stat_free(lnk);
-      break;
+  if (frametables->data == table) {
+    lnk = frametables;
+    frametables = frametables->next;
+    caml_stat_free(lnk);
+  } else {
+    iter_list(frametables->next,lnk) {
+      if(lnk->data == table) {
+	previous->next = lnk->next;
```

Comment:
I think this is mixing tabs and spaces for indentation, or at least it's rendering weird when I view it.

Reply:
Should it be all spaces?